content/docs/introduction/comparison.md

```diff
@@ -132,14 +132,13 @@ boundaries like products, services, datacenters, or similar aspects.
 Independent servers (which can be run redundantly in parallel) may also give
 you better reliability and failure isolation.
 
-Kapacitor currently has no [built-in distributed/redundant
+Kapacitor's open-source release currently has no [built-in distributed/redundant
 options](https://github.com/influxdata/kapacitor/issues/277) for rules,
-alerting, or notifications. Prometheus and the Alertmanager by contrast offer a
-redundant option via running redundant replicas of Prometheus and using the
-Alertmanager's [High
-Availability](https://github.com/prometheus/alertmanager#high-availability)
-mode. In addition, Kapacitor can be scaled via manual sharding by the user,
-similar to Prometheus itself.
+alerting, or notifications.  Prometheus and the Alertmanager by contrast offer a 
+redundant option via running redundant replicas of Prometheus and using the 
+Alertmanager's [High Availability](https://github.com/prometheus/alertmanager#high-availability)
+mode. In addition, Kapacitor can be scaled via manual sharding by the user, similar 
+to Prometheus itself. 
```

Comment:
You've removed the mention of the commercial Kapacitor, which I thought was the point of this PR.

Reply:
Well, the point of the PR was to clear up the "kapacitor doesn't support HA" line...  As it does now, with the commercial version at least.  

I just re-added the link for Enterprise Kapacitor and Enterprise Influx at the bottom of the summery section... 
Does this look better?